third-party-brushes/shBrushTs.js

```diff
@@ -0,0 +1,38 @@
+SyntaxHighlighter.brushes.ts = function () {
```

Comment:
This probably needs to mention https://github.com/syntaxhighlighter/brush-typescript (MIT licensed)